ui/dashboards/src/components/Variables/VariableList.tsx

```diff
@@ -26,13 +27,14 @@ interface TemplateVariableListProps {
 
 export function TemplateVariableList(props: TemplateVariableListProps) {
   const [isEditing, setIsEditing] = useState(false);
+  const [isPin, setIsPin] = useState(props.variableIsSticky);
```

Comment:
Super nit: It's generally good practice to name props in a way that makes it clear that the component is controlled vs [uncontrolled](https://reactjs.org/docs/uncontrolled-components.html).

For example, the native input uses `default` prefix when the `value` of the input is uncontrolled.
```tsx
<input value="foo" /> // controlled
<input defaultValue="foo" /> // uncontrolled
```

Maybe `variableIsSticky` might be better as `initialVariableIsSticky`

Reply:
interesting ! Makes sense I will change that.  Thanks !